controllers/serverTableController.js

```diff
@@ -1,27 +1,65 @@
-var app = angular.module('pokemonDraft', []);
-app.controller('serverTableController', function($scope, $http) {
+var app = angular.module('pokemonDraft', ['ui.bootstrap.modal', 'ui.bootstrap.tpls']);
+app.controller('serverTableController', function($scope, $http, $modal) {
+	// server list
```

Comment:
If possible use variable names to describe what your doing over a comment, only comment if the intent is not immediatly clear.

If hostedGames is the server list, mase well just call it that. Although there will be multiple games per server, so im not sure what the intent of this is here? (im being picky to give as much feedback as possible)